doc/Journal.tex

```diff
@@ -220,143 +220,113 @@ \section{Background}
-First, $n$ $k$ values are chosen.  
-These values are then placed in a stiffness matrix corresponding to the coupled spring/mass problem shown in Fig.\ref{fig:stiff}.  
+So, the optimized design objective is to create a design with the least amount of linear dependence.  
+The three following methods are tailored to create designs for identifying both $\theta$ and $\phi$ with a low linear dependence.  
+The eigenvector approach solves a mass normalized eigenvalue problem, the binary method uses patterns of ones and zeros to represent the geometry, and a modification of the Hadamard approach used for rotating encoding masks is explored.
+
+\subsubsection{Eigenvector Approach}
+The eigenvector approach creates a basis set, which may be used to define the geometry space.  
+% The replication of the k variable is unfortunate.
+First, $n$ $k$ values are chosen, where $k$ is the spring constant.  
+These values are then placed in a stiffness matrix corresponding to the coupled spring-mass problem shown in Fig.\ref{fig:stiff}.  
 The mass is assumed to be normalized.  
-This approach assumes there is additional coupling between nearby springs.  
-This choice represents the spacial coupling among nearby $\phi$ positions on the RSM.
-Other methods of coupling can be introduced if desired.
+This approach assumes there is additional coupling between nearby springs, which represents the spatial coupling among nearby $\phi$ positions on the RSM.
+Other methods of coupling could be introduced if desired.
 
 \begin{figure}[ht!]
 \includegraphics[width={3.0in}]{../figs/MassSys.pdf}
 \centering
-\caption{Equivalent spring/mass system used to generate the eigenvectors}
+\caption{Equivalent spring-mass system used to generate the eigenvectors.}
 \label{fig:stiff}
 \end{figure}
 
 Note that the values chosen do not need to represent physical systems (e.g. negative stiffness values are acceptable).  
+% This statement should be expounded upon to at the very least say which systems were explored.  Why were some of the other systems not included?
 Other coupling systems were evaluated.  
-However the best results corresponding to the system show in Fig.~\ref{fig:stiff}.  
+However, the best results corresponding to the system show in Fig.~\ref{fig:stiff}.  
 
-Then, an eigenvalue problem is solved resulting in $n$ orthonormal eigenvectors used to represent the geometry for initial position $\theta=0$ (and all initial $phi$ positions).  
+Then, an eigenvalue problem is solved resulting in $n$ orthonormal eigenvectors used to represent the geometry for initial position $\theta=0$ and all initial $phi$ positions.  
 As the first vector tends to be planar motion (aka a cyclical vector) it is not chosen.  
 This elimination results in a matrix formed by eigenvectors $2...m+1$.  
+% I am missing something on the transition from the previous sentence to the next that could use some clarification.
```

Comment:
The first vector tends to be symmetric since it represents translation motion where all the entries in the vector are the same.  Thus, I skip the first vector as I know that cyclic symmetry is not desired for the RSM DRCs.

Reply:
Got it